.github/workflows/rust.yml

```
@@ -8,7 +8,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
-  RUST_VERSION: 1.76.0
+  RUST_VERSION: 1.77.0
```

```suggestion
  RUST_VERSION: 1.79.0
```

Looks like this is why the CI failed.

---

Yeah, I misread it for requiring 1.77 - not 1.79.